apps/docs/content/components/Grid/index.mdx

```diff
@@ -0,0 +1,286 @@
+
+Use the grid for elements with related content or similar purposes. E.g. A list of [cards](/components/Card) or a "contact us" form.
+
+In most cases, we recommend using the same column span for each element to create a consistent layout.
+
+[^1]: The grid component is implemented using the [CSS Grid Layout](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Grid_Layout) specification.
+
+<DoDontContainer>
+  <Do>
+    <img
+      src="https://github.com/primer/brand/assets/912236/b4870b2e-5cf0-440c-9f51-0b1f81258b9b"
+      alt=""
+    />
+    <Caption>
+      Use the same column span for each content unit to create a consistent
+      layout and spacing.
+    </Caption>
+  </Do>
+  <Dont>
+    <img
+      src="https://github.com/primer/brand/assets/912236/0703ba43-7f47-4ad1-bc1d-020bc095e362"
+      alt=""
+    />
+    <Caption>
+      Don't use different column spans when grouping the same element type.
+    </Caption>
+  </Dont>
+</DoDontContainer>
+
+In some cases, you may need specific amount of columns or rows instead of an even distribution. Use different column spans when creating a visual hierarchy between items or when defining a page or section layout structures. E.g. A section with an [image](/components/Image) and [text](/components/text), a form with inputs and text areas, or a layout with main content and a sidebar.
+
+![](https://github.com/primer/brand/assets/912236/bc442c41-3546-4341-bab3-1c649ccfe7d2)
+
+<DoDontContainer stacked>
+  <Do>
+    <img
+      src="https://github.com/primer/brand/assets/912236/83e4eedf-5cc2-4dcc-a9dd-48d563744dd2"
+      alt=""
+    />
+    <Caption>
+      Use different column spans when you want to create a visual hierarchy
+      between different elements.
+    </Caption>
+  </Do>
+  <Dont>
+    <img
+      src="https://github.com/primer/brand/assets/912236/911a6b4b-791a-4186-8079-6296e17c65d0"
+      alt=""
+    />
+    <Caption>
+      Don't use equal column spans when the content is not related or the
+      elements are different.
+    </Caption>
+  </Dont>
+</DoDontContainer>
+
+### Responsive
+
+The grid component provides control over the number of columns at each breakpoint. For most of the cases, we recommend using the same column span on each child at each breakpoint to create a consistent layout and spacing in all screen sizes.
+
+For example, a group of cards might span 3 columns on a extra large viewport, 4 columns on large viewports, 6 on medium and 12 columns on small viewports.
+
+<div>
+  <CustomVideoPlayer
+    width="100%"
+    src="https://github.com/primer/brand/assets/912236/a99beba9-b22f-4e2b-a5fa-66c2db246965"
+  />
+  <Caption>
+    Use the same column span for each content unit at each breakpoint to create
+    visually consistent layouts.
+  </Caption>
+</div>
+
+When using even column spans, we recommend to span each item full width on small viewports and stacking the items vertically to ensure that the content is readable and accessible on small screens.
```

Comment:
Could be a neat opportunity to link the user directly to the section and examples about responsive behaviour?https://primer-63f891a15f-26139705.drafts.github.io/components/Grid/react#responsive-behavior

Reply:
I think that section is more a "how-to-implement" example rather than a design/usage guideline. I'd keep those two aspects separate by our tabs rather than linking them back and forth